trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/chandler/FunctionHandler.java

```diff
@@ -880,7 +909,6 @@ private void handleFunctionsInParams(final IDispatcher main, final ILocation loc
 			assert false : "are we missing a type of function declarator??";
 		}
 
-		assert inparamVarListArray.length == paramDecs.length;

```

Comment:
Would it make sense to keep this assert, but modify it so it also covers the case with varargs?

Reply:
We could make it `>=` instead, but if the assertion is violated then, we get an exception in the loop afterwards anyway.

Reply:
Ok, I'll leave that up to you